README.md

```diff
@@ -41,13 +41,3 @@ All the installations, configuration, and usage instructions are available in th
 + Automatic uploads tagging system.
 + Tag uploads with custom tags for categorization.
 + ... and more.
-
-
-## Security Vulnerabilities
-
-If you discover a security vulnerability within XBackBone, please send an e-mail to Sergio at sergio@brighenti.me. All security vulnerabilities will be promptly addressed.
-
-## License
-This software is licensed under the <a href="https://choosealicense.com/licenses/agpl-3.0/">GNU Affero General Public License v3.0</a>, available in this repository.
-As a "copyright notice" it is sufficient to keep the small footer at the bottom of the page, also to help other people to learn about this project!
-
```

Comment:
<!-- Issues represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 issue:** ‏Why thi got removed?

Reply:
I explain it in the commit already, but I will do it again it you mentioned the LICENSE and SECURITY in the README.me which you need to do that it's already a file and people will see it if they scroll through the source code